src/nix/flake-init.md

```diff
@@ -45,6 +49,10 @@ outputs = { self }: {
   templates.rust = {
     path = ./rust;
     description = "A simple Rust/Cargo project";
+    welcomeText = ''
+      You've created a simple Rust/Cargo template.
+      Visit https://www.rust-lang.org/ for more info.
```

Comment:
I don't think this is a useful link. It should point to a Nix-specific document.

I understand that it's just an example, but we shouldn't set a wrong precedent.